x-pack/plugins/observability_solution/dataset_quality/server/routes/data_streams/routes.ts

```diff
@@ -13,14 +13,15 @@ import {
   DataStreamStat,
   DegradedDocs,
   Integration,
+  IntegrationDashboards,
 } from '../../../common/api_types';
 import { rangeRt, typeRt } from '../../types/default_api_types';
 import { createDatasetQualityServerRoute } from '../create_datasets_quality_server_route';
 import { getDataStreamDetails } from './get_data_stream_details';
 import { getDataStreams } from './get_data_streams';
 import { getDataStreamsStats } from './get_data_streams_stats';
 import { getDegradedDocsPaginated } from './get_degraded_docs';
-import { getIntegrations } from './get_integrations';
+import { getIntegrationDashboards, getIntegrations } from './get_integration_dashboards';
```

Comment:
[nit] What about leaving the file name as it was before? `get_integrations` I think suits better